### PR TITLE
Implement model validation pipeline

### PR DESCRIPTION
**Pull request by VitaKotenko**
dev
## Summary of issue

ToDo

## Summary of change

ToDo

## Comments

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=50%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  PR meets all conventions

### CODE REVIEW

**Comment by VitaKotenko** [comment]
VallidationError(failures) gets List<ValidationFailure>failures.
ValidationFailure object has next properties:
public string PropertyName { get; set; }
public string ErrorMessage { get; set; }
public object AttemptedValue { get; set; }
public object CustomState { get; set; }
public Severity Severity { get; set; } = Severity.Error;
public string ErrorCode { get; set; }

Class ValidationException has property Errors type of IEnumerable<ValidationFailure>, which contains the list of validation errors.
When constructor is called, it will generate message by the BuildErrorMessage(errors) method from Exception class.

Also I added links on the [VallidationError ](https://github.com/FluentValidation/FluentValidation/blob/main/src/FluentValidation/ValidationException.cs) and [ValidationFailure](https://github.com/FluentValidation/FluentValidation/blob/main/src/FluentValidation/Results/ValidationFailure.cs)

File: Streetcode/Streetcode.WebApi/Program.cs [hunk 1]
```diff
@@ -13,6 +17,8 @@
 builder.Services.ConfigurePayment(builder);
 builder.Services.ConfigureInstagram(builder);
 builder.Services.ConfigureSerilog(builder);
+builder.Services.AddModelValidationServices();
```
Comment (jimmy2jk) [inline Streetcode/Streetcode.WebApi/Program.cs:20]: move this call into `ServiceCollectionExtensions.AddCustomServices` method

**Review by jimmy2jk — CHANGES_REQUESTED** [review]
Move call of Validator registration method to AddCustomServices method. (Near AddAutoMapper and AddMediatR)
Everything else seems fine!

File: Streetcode/Streetcode.WebApi/Program.cs (hunk 1 above)
Comment (VitaKotenko) [reply Streetcode/Streetcode.WebApi/Program.cs]: Done

**Review by MishaHMK — APPROVED** [review]

**Review by jimmy2jk — APPROVED** [review]

**Pull request merged**